Fixing 16+ COL Keyboards / Added Mask

IBNobody opened the pull request:
Fixed some issues with large keyboards.

Added a mask option to the matrix_get_row function to ignore matrix rows that have keys that are stuck down on purpose.

jackhumbert commented:
Awesome! It looks like some other files were changed too - were these intentional/necessary?

IBNobody commented:
The Writer's Block code was checked in with incorrect line endings, and I
was the last person to sync the repo. Thus, my git fixed the line endings
automatically.

I also fixed the debug magic key to work as intended: enable debug messages
or disable debug messages, keyboard messages, matrix messages, and mouse
messages. Thus reverses/revises my last change to it.

Finally,I added the EXTRAFLAGS comment to the readme because I kept
forgetting that flag.

On Tue, Oct 25, 2016, 11:51 PM Jack Humbert <notifications@github.com>
wrote:

> Awesome! It looks like some other files were changed too - were these
> intentional/necessary?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/jackhumbert/qmk_firmware/pull/846#issuecomment-256249000>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AFl50Fq5yC2sjdt4mx98_IufONjzXSAzks5q3txEgaJpZM4KeYHQ>
> .
>

IBNobody commented:
Closing PR and will resubmit once the matrix debounce code is corrected.
